src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -461,7 +461,7 @@ private async Task AddHttp11ConnectionAsync(HttpRequestMessage request)
             {
                 try
                 {
-                    connection = await CreateHttp11ConnectionAsync(request, false, cts.Token).ConfigureAwait(false);
+                    connection = await CreateHttp11ConnectionAsync(request, true, cts.Token).ConfigureAwait(false);
```

Comment:
How would this work for the `Sync` HttpClient? I did not see any ` AddHttp11Connection` counterpart. 
This is not my area but I was under impression that we want to do _alway_ sync operations for the sync client and `async` for "normal" case. Having the value fixed makes me wonder....

Reply:
The only call site to this method is just queueing it via a Task.Run, so there won't ever be a sync caller.

However, this does raise the point that I think Geoff's previous change around connection pooling does mean that we're never creating connections synchronously when we used to, which has a couple ramifications we should consider.  First, it means every socket on Unix will be in a non-blocking state, which will make the synchronous operations performed on that connection more expensive.  Second, it means that some number of sync requests will be done sync-over-async as they synchronously block waiting for a connection to be available, whereas previously they would have actually been creating the connection synchronously.